packages/core/src/SpringValue.test.ts

```diff
@@ -467,6 +467,35 @@ function describeConfigProp() {
         expect(global.countBounces(spring)).toBeGreaterThan(0)
       })
     })
+    describe('the "precision" prop', () => {
+      describe('stops decay animation when step difference is less than precision', () => {
```

Comment:
Would you be able to add a test around it effecting regular springs without `decay` applied?

Reply:
Yep! Added a test for spring easing in commit cd43a70